docs/guides/admin/docs/workflowoperationhandlers/video-grid-woh.md

```diff
@@ -0,0 +1,64 @@
+# VideoGridWorkflowOperationHandler
+
+## Description
+
+The VideoGridWorkflowOperationHandler offers a way to combine several, partially simultaneously
+playing videos into a single video file. For example, the webcam feeds during a video conference
+can be combined by this WOH. The resulting video puts each input video on a grid that dynamically
+resizes based on the number of inputs videos currently active. Which input video is active when
+is defined by through a SMIL catalogue from e.g. a partial ingest.
+
+If the SMIL defines a section where there are no videos active, the background color will be shown
+instead for the duration of the section. This also holds true for potentially empty beginning and end 
+sections, ensuring that a final single video is as long as the overall duration defined in the SMIL 
+(e.g. if the first input video becomes active at 30 seconds, the first generated output is a 30 second
+long video of the background color). The background color is also shown whenever the input videos cannot
+fully fill up the available space.
+
+This WOH relies on the inspect service for enriching generated, temporary video files with metadata.
+Furthermore, it relies on the composers concat service to combine temporary video files into
+a single output file.
+
+## Parameter Table
+
+|configuration keys | example                     | description                                                         |
+|-------------------|-----------------------------|---------------------------------------------------------------------|
+|source-flavor      | presenter/source            | Flavor containing all the video tracks to be combined.                              |
```

Comment:
I see no reason not to allow for more than one flavor, so sure, will do.

Reply:
It was only a question because it is not so explicit in the docs and is natural to think that you can video-grid all the videos. Ex: To do a video grid with the presentation. 

For time constrains for this version is ok if you left with only one flavor, you can add for more flavors